restapi/src/main/java/io/stargate/web/models/Datacenter.java

```diff
@@ -18,12 +18,14 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Datacenter {
   String name;
   int replicas;
 
+  @ApiModelProperty(required = true, value = "")
```

Comment:
Should these fields be documented? Esp. "replicas".

Reply:
FYI this is a response object. The input is a map since it can be either

```json
{ "name": "killrvideo", "replicas": 1}
```

or
```json
{
  "name": "killrvideo",
   "datacenters":
      [
         { "name": "dc1", "replicas": 3 },
         { "name": "dc2", "replicas": 3 },
      ],
}
```